zapisy/apps/defects/assets/DefectList.vue

```diff
@@ -70,17 +70,26 @@ export default class DefectList extends Vue {
   select(event) {
     // @ts-ignore
     !event.currentTarget.classList.toggle("selected");
-    if (document.getElementsByClassName("selected").length > 0) {
-      // @ts-ignore
-      document.getElementById("print-form-button").value = "Drukuj zaznaczone";
+    let selected_defects = document.getElementsByClassName("selected");
+    let print_button = document.getElementById("print-button");
```

Comment:
```suggestion
    let print_button = document.getElementById("print-button")!;
```
My chyba nie przewidujemy sytuacji, w której elementu o takim ID nie ma, prawda? Jak użyjemy operatora _non-null assertion_ to nie będzie trzeba tego niże sprawdzać.